tools/rebuild-image

```diff
@@ -40,13 +40,27 @@ commit_timestamp() {
 	git show -s --format=%ct "$rev"
 }
 
+# is the SHA1 actually present in the repo?
+# it could be it isn't, e.g. after a force push
+commit_valid() {
+	local rev=$1
+	git rev-parse --quiet --verify "$rev^{commit}"
```

Comment:
Maybe add `>/dev/null` at the end? The `--quiet` option prevents the printing of the error, but in case of the correct rev, it will print it in its full form.